src/coresyf_image_crop/coresyf_image_crop.py

```diff
@@ -69,6 +68,47 @@ def get_shapefile_crs(data_source):
     return epsg_code
 
 
+def get_raster_crs(data_source):
+    '''
+    Retrieves the ESPG Code of the raster CRS (Coordinate Reference System).
+
+    data_source: must be the result of GDAL Open applied to the respective GDAL
+    driver.
+    '''
+    try:
+        projection = data_source.GetProjectionRef()
+        ident = Sridentify(prj=projection)
+        epsg_code = ident.get_epsg()
+    except Exception as epsg_exception:
```

Comment:
What are the exceptions expected here? It is not a good write generic except block catching every possible exception in a function. The only place where such would be acceptable is in the script main code, where error messages can be written to stdout, like is done in this block. In functions we should handle only the specific exceptions that the function expects and knows exactly how to handle. 
There are at least two main reasons for this:

1. The block can be handling an exception was not predicted in a way it is not appropriate for such exception.
Indeed, it happens here. If an object which is not a GDAL Driver is passed to the function an `AttributeError` is raised in line `79`, and still the message will be _Unable to get ESPG code of raster_, when there is no raster at all. 

2. This prevents the caller to handle the exception by itself. For example, if we want to define a default EPSG code for the case the raster EPSG code is not found, we couldn't call this function as is since the program would exit without giving opportunity for such. 

I think here it would better if we did not handle the exceptions at all.